### PR TITLE
Extract shared code to util repo

### PR DESCRIPTION
**Pull request by jordanCain**

### CODE REVIEW

File: tests/FVTTests/test/filterParametersView/testFilterParameterView.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+ */
+
 /* eslint-disable no-unused-expressions */
 const { By } = require('selenium-webdriver');
 const { expect } = require('chai');
 const chai = require('chai');
 chai.use(require('chai-things'));
 require('geckodriver');
 
+const { getDriver, checkDriver } = require('explorer-fvt-utilities');
```
Comment (NakulManchanda) [inline tests/FVTTests/test/filterParametersView/testFilterParameterView.js]: require add testElementAppearsXTimesByCSS

File: tests/FVTTests/test/homeView/testHomeView.js.js [hunk 2]
```diff
@@ -8,6 +18,12 @@ require('geckodriver');
 const {
     getDriver,
     checkDriver,
+    testElementAppearsXTimesById,
+    testWindowHeightChangeForcesComponentHeightChange,
+    testTextInputFieldCanBeModified,
+} = require('explorer-fvt-utilities');
```
[the remark this answers is not in the document]
Comment (jordanCain) [reply tests/FVTTests/test/homeView/testHomeView.js.js]: Need to figure out the dependency structure here, the test utilities module currently references geckodriver as a peer dependency, I don't know if the test utilities should care about the browser version under test

[the remark this answers is not in the document]
Comment (NakulManchanda) [reply tests/FVTTests/test/homeView/testHomeView.js.js]: We should make use of shared repo 'explorer-fvt-utilities'  to control version of web driver so all explorers jes, mvs can make use of same version of webdriver. 
i believe it should be responsibility of repo managing and initialising driver.

[the remark this answers is not in the document]
Comment (NakulManchanda) [reply tests/FVTTests/test/homeView/testHomeView.js.js]: Additionally, look at PR
https://github.com/zowe/explorer-jes/pull/153

When we add chrome - we have to add yet another line in all test suites - require('chrome')
Another entry for edge and safari, we should require it once only. That too in all explorers FVT tests files separately.

Lets make use of shared repo to maintain version of wedrivers.

**Comment by NakulManchanda** [comment]
I believe we can still improve the location of driver, will create separate PR to relocate that.
Its a good start. Nice work, thanks!

Will rework PR 153 with new shared repo. Thanks!

File: tests/FVTTests/test/homeView/testHomeView.js.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (NakulManchanda) [reply tests/FVTTests/test/homeView/testHomeView.js.js]: @jordanCain  Approving this. Will rework PR 153 with new shared repo. Thanks!

**Pull request merged**